<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPasswdPage</class>
 <widget class="QWidget" name="ForgetPasswdPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="whiteBlockWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>500</y>
     <width>384</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 27px;</string>
   </property>
   <widget class="QLineEdit" name="lineEditAuthCode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>321</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>请输入验证码</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="whiteBlockWiget_1" native="true">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>430</y>
     <width>384</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 27px;</string>
   </property>
   <widget class="QLineEdit" name="lineEditTel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>321</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
    <property name="placeholderText">
     <string>请输入手机号</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>67</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/back.png);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>783</x>
     <y>388</y>
     <width>353</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>验证码已发送至189****6666，请输入相应验证码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>848</x>
     <y>586</y>
     <width>224</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 76, 76);</string>
   </property>
   <property name="text">
    <string>请先检查您的网络</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1166</x>
     <y>500</y>
     <width>132</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 135, 255);
color: rgb(255, 255, 255);
border-radius: 27px;
border:none;</string>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEth">
   <property name="geometry">
    <rect>
     <x>1811</x>
     <y>981</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/eth.png);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
